bayesian/utils.py

```diff
@@ -103,6 +103,21 @@ def server_create_component_bookkeeping(ecosystem, name, version, user_profile):
     return server_run_flow('componentApiFlow', args)
 
 
+def server_create_component_bookkeeping_post(ecosystem, header):
```

Comment:
Can this be called create_component_analysis_bookkeeping. I dont think the post is required. Like I said in the worker PR if the user_email and user_profile are always 'None' there is no point sending it.

Reply:
Yes,I will change accordingly.